geopandas/geodataframe.py

```diff
@@ -431,7 +431,7 @@ def rename_geometry(self, col, inplace=False):
         else:
             if not inplace:
                 return self.rename(columns={geometry_col: col}).set_geometry(
-                    col, inplace
+                    col, inplace=inplace
```

Comment:
This was (accidentally, I suppose) passed as the `drop` keyword because of using a positional keyword, and therefore this was triggering a whole bunch of warnings that the user shouldn't see.

Given that this is the case of `inplace=False`, we were essentially passing `drop=False` (the default), so no longer passing it should not have any effect

Reply:
This makes sense, agree it looks like the change won't change behaviour, only fix warnings.